app/controllers/api/v2/registration_commands_controller.rb

```diff
@@ -15,6 +15,8 @@ class RegistrationCommandsController < V2::BaseController
         param :setup_remote_execution, :bool, desc: N_("Set 'host_registration_remote_execution' parameter for the host. If it is set to true, SSH keys will be installed on the host")
         param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours)")
         param :insecure, :bool, desc: N_("Enable insecure argument for the initial curl")
+        param :repo, String, desc: N_("Repository URL / details, for example for Debian OS family: 'deb http://deb.example.com/ buster 1.0', for Red Hat OS family: 'http://rpm.example.com/'")
```

Comment:
just a tiny suggestion for the rpm example, how about `http://yum.theforeman.org/client/latest/el8/x86_64/` that suggests the common structure of version/arch/ and demonstrates the purpose e.g. is someone needs some client tooling for the actual registration